web/src/popovers.js

```diff
@@ -31,8 +31,17 @@ export function any_active() {
 export function hide_all_except_sidebars(opts) {
     $(".has_popover").removeClass("has_popover has_actions_popover has_emoji_popover");
     if (!opts || !opts.not_hide_tippy_instances) {
-        // hideAll hides all tippy instances (tooltips and popovers).
-        hideAll();
+        if (opts && opts.not_hide_tippy_dropdowns) {
+            // don't hide personal menu dropdown
+            // TODO: after we add tippy gear menu dropdown, we should add a function
+            // that gets the current open tippy dropdown instance and close it.
+            hideAll({
+                exclude: popover_menus.get_personal_menu_instance(),
+            });
+        } else {
+            // hideAll hides all tippy instances (tooltips and popovers).
+            hideAll();
+        }
```

Comment:
Let's drop this commit and leave this question for follow-ups; whether this new menu hides or not on scroll is not important, and https://github.com/zulip/zulip/pull/26884 is going to merge conflict from this and might happen to make this work the way we want it anyway.

Reply:
Done